### ISSUE TITLE
center the image

### ISSUE THREAD

**Issue by orlandombaa** [kylebarron/keplergl-quickvis#8]
Hello 

I have a question. It is possible to center the image according to the data? It always appears in San Fransisco.

Thank you!

**Comment by kylebarron** [comment]
I think this is a regression from using one of the newer versions of `keplergl`. In older versions as of a year or two ago, the data was always centered. I'm not sure if that's a bug or intentional change in upstream `keplergl` or something I need to fix in here.

Pull requests welcome!

**Comment by cholmes** [comment]
Recently found this tool and it is exactly what I need, if only it would automatically center on the data. 

Fixing the bug is beyond my current python + js abilities, but I was able to pinpoint the version change where this broke. If I change the keplergl requirement in requirements.txt to 0.1.2 then it all works great. If it's 0.2.0 then it doesn't center. 

I tried to dig into the changelogs to see if it was an intentional change, and didn't see anything in kepler.gl itself, and couldn't find changelogs for the kepler.gl python library. 

It does seem like it should be possible to 'fix' this in the cli library, just get the bounds of the data when it loads, and then set the map center (likely loading a new 'config' with it).
